### ISSUE TITLE
Material node values do not initially reflect the displayed control/slider values.

### ISSUE THREAD

**Issue by PhysicsTeacher1** [AsoboStudio/glTF-Blender-IO-MSFS#107]
### Current Behavior

When a new material is created, the Material Properties sliders and controls do not reflect the actual default values of the nodes in the created material node tree.  This appears to be a problem with initialization when a new material is created.  A work-around is to manipulate the control slider/value once, and that will make the control/node values equal.

### Expected Behavior

The parameters of relevant nodes created by the MSFS Standard Material should reflect the values set in the node tree of the instance created.

### Steps To Reproduce

1. Create new object
 2. Create a new texture for the object.  Assume a simple Base Color to start; this invokes the default white value.
 3. In the material stack, set MSFS Material Params to Standard.
 4. Without doing anything else, especially changing the Base Color, switch to a Blender view of the Shading Editor, the shader tree generated by the creation of the new material.
 5. In the Shader Editor, navigate to the Base Color RGB node, near the top of the screen.
 6. Click the flippy arrow of that node to expand the node.
 7. Note that that node is defaulted to a gray color while the Base Color control in the properties panel is pure white (1.00).
 8. In the Properties panel, click the Base Color control to get the Color Picker dialog.  Note that the Color Picker dialog shows full white.
 9. With your mouse, move the gray scale slider on the right of the Color Picker dialog up and down a little once.
 10. Note the value in the node controlled by the Color Picker dialog now repeats the Base Color control.

### Environment

```markdown
- OS: Windows 10 Home
- Blender:3.1
- glTF-Blender-IO-MSFS:1.1.3
```

### Anything else?

Best guess as to the solution for this problem:
 In the __init__ portion of the material class instance constructor, there should be a segment that scans the visible default settings of the material properties controls to make the nodes reflect the visible settings.  This problem is apparent in many of the control/node pairings.
 Minor Consideration:
  Controls like Metallic Factor, Roughness Factor, Normal Scale, Emissive RGB and the like should be defaulted to reasonable values if they are used in the material.  The default set in the Principled BSDF Shader in Blender would be a reasonable set.
  
 Summary comment:
  I have been using this new exporter extensively for the past week, and I find it a very agreeable tool.  I am currently recompiling many of my projects using it.  My thanks to all the developers for this well-organized tool.  I await the resolution of the order of LOD keys in the .xml file.  Thank you all for the work on this tool.

**Comment by Mikeaat** [comment]
Hey i think this is also the same like #89 but better detailed.

**Comment by PhysicsTeacher1** [comment]
I actually have it "fixed" and am testing.  You'll have to take care of the changes, as I am not qualified to get anywhere near a code repository.  I'll detail it here for you.  The fix is not optimal, but it will point to the problem.  Help!  I'm being dragged into the jungle by a large snake!

**Comment by PhysicsTeacher1** [comment]
I just looked at the changes in 1.1.4 released a few hours ago, and they are a better fix than mine.  All I did yesterday was comment out the attempt to set the color mode to Non-Color.  That's where the update was hanging and not being caught.  I found that problem using the Blender Console.  Thanks to lpieriabella for this fix that incorporates several other fixes also.  The fix should be solid.
